src/pal/inc/pal.h

```diff
@@ -68,6 +68,12 @@ extern "C" {
 #include <pal_error.h>
 #include <pal_mstypes.h>
 
+// Native system libray handle.
+// In Windows, NATIVE_LIBRARY_HANDLE is the same as HMODULE.
+// On Unix systems, NATIVE_LIBRARY_HANDLE type represents a library handle not registered with the PAL.
+// To get a HMODULE on Unix, call PAL_RegisterLibraryDirect() on a NATIVE_LIBRARY_HANDLE.
+typedef HANDLE NATIVE_LIBRARY_HANDLE;
```

Comment:
A nit - the dlopen returns void*, so it would be cleaner to define it that way.

Reply:
Also, I cannot see the type definition for Windows. The pal.h is included for Unix only.

Reply:
I added a definition in `common.h`, for the `!FEATURE_PAL` case Thanks

Reply:
inc/palclr.h would be a better place for it. It is where all similar definitions are.

Reply:
Thanks, I changed the Windows definition to be in palclr.h via palclr_win.h